### ISSUE TITLE
Things left to patternify

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/Tidal#361]
Still some things hiding..

* `scale`
* `discretise` (had problems with this before, hopefully doable in the new tidal)

**Comment by msp** [comment]
I've seen a few references to this kind of thing in the git history. If these follow a similar solution would they be suitable for a Haskell newbie to pick up? Happy to try if so!

**Comment by yaxu** [comment]
@msp yes should be straightforward.. A case of using `tParam` (or `tParam2` etc if there is more than one parameter to patternify). I did have a problem with using `tParam` with `discretise` before the refactor but hopefully it will work properly now..

**Comment by msp** [comment]
Cool, I'll take a look.

**Comment by msp** [comment]
Opened #366

**Comment by msp** [comment]
Is there anything left to do with `discretise` after 
https://github.com/tidalcycles/Tidal/commit/d026f4a93aee7a278bfab5e836447c2d9a2177e9 & https://github.com/tidalcycles/Tidal/commit/31e5c97857938aea534441e49a8c16527bfc484b ?
